Crash on send to Top.gg if API is busy

Crash on send to Top.gg if API is busy, I would also expect this to happen if a rate limit is applied.

```
Environment: Docker
Bot version: 1.0.3-1.0.4 (all?)
Self-hosted: ❌ 
```


<details>
<summary>Logs</summary>
Post "https://top.gg/api/bots/{id}/stats": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x6b121c]
goroutine 55 [running]:
github.com/lucxjo/diru/utils.SendTopggData({0xc000150140, 0x9b}, {0xc000018168, 0x12}, 0x1, {0xc000116300, 0x3b})
	/app/utils/topgg.go:31 +0x3dc
main.main.func1()
	/app/diru.go:33 +0x171
created by github.com/andersfylling/disgord.(*rdyCtrl).OnRemove
	/go/pkg/mod/github.com/andersfylling/disgord@v0.32.3/reactor.go:403 +0x8e
</details>

--- a/utils/topgg.go
+++ b/utils/topgg.go
@@ -10,22 +10,33 @@ import (
 
 func SendTopggData(token string, botId string, shards uint, dctoken string) {
 	httpClient := &http.Client{
-		Timeout: 10 * time.Second,
+		Timeout: 60 * time.Second,
 	}
 
-	reqBody, _ := json.Marshal(map[string]uint{
+	reqBody, err := json.Marshal(map[string]uint{
 		"server_count": uint(GetConnectedServerCount(dctoken)),
 		"shard_count": shards,
 	})
 
-	req, _ := http.NewRequest("POST", "https://top.gg/api/bots/" + botId + "/stats", bytes.NewBuffer(reqBody))
+	if err != nil {
+		fmt.Println("Error marshalling top.gg data:", err)
+		return
+	}
+
+	req, err := http.NewRequest("POST", "https://top.gg/api/bots/" + botId + "/stats", bytes.NewBuffer(reqBody))
 	req.Header.Set("Content-Type", "application/json")
 	req.Header.Add("Authorization", token)
+
+	if err != nil {
+		fmt.Println("Error starting request, top.gg data:", err)
+		return
+	}
 
 	resp, err := httpClient.Do(req)
 
 	if err != nil {
 		fmt.Println(err)
+		return
 	}
 
 	defer resp.Body.Close()

